### ISSUE TITLE
Question: Hunyuan : How are steps per epoch calculated?

### ISSUE THREAD

**Issue by caniyabanci76** [tdrussell/diffusion-pipe#57]
For training Hunyuan with images, how are steps per epoch calculated?  I'm figuring that it's to do with repeats * number of training images but it isn't quite right

**Comment by alisson-anjos** [comment]
(Size of Dataset * Num Repeats) / (Batch Size * Gradient Accumulation Steps) * Epochs
